### DISCUSSION TITLE
I need help with slick slider, when initial loading in phone it shows 6 slides whereas in breakpoints it has setting of 2 slides in mobile preview.

### DISCUSSION THREAD

**Discussion by naveenkhatanagit** [vercel/next.js#50211] [Help]
### Summary

I need help with slick slider, when initial loading in phone it shows 6 slides whereas in breakpoints it has setting of 2 slides in mobile preview.

**Comment by icyJoseph** [comment]
How is "mobile" being calculated? CSS? or is it JavaScript mediaQuery object? 

The issue might be that the pre-rendered version (on the server) defaults to wide screen.
